slices/libmemcached11t64.yaml

```diff
@@ -0,0 +1,19 @@
+package: libmemcached11t64 
+
+essential:
+  - libmemcached11t64_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libhashkit2t64_libs
+      - libsasl2-2_libs
+      - libstdc++6_libs
+    contents: 
+      /usr/lib/*-linux-gnu/libmemcached.so.11*:
+  copyright:
+    contents: 
+      /usr/share/doc/libmemcached11t64/copyright:
+    
```

Comment:
Let's remove the blank line at the end of this file.

Also, some of the files you have been creating lack a newline (`\n`) at the very end of the file (as denoted by the red minus sign in a circle above).
Can you check your IDE config and update it so that it will include a newline (`\n`) at the end of files?